cmd/config/internal/commands/cmdset.go

```diff
@@ -75,6 +75,11 @@ func initSetterVersion(c *cobra.Command, args []string) error {
 		setterVersion = "v1"
 	}
 
+	if setterVersion == "v1" {
+		fmt.Printf("\nWarning: You are using older version of setters which will be deprecated soon, " +
```

Comment:
As far as I can see, we are not actually adding the pkg fix command in this PR. We should add this warning when we have the actual command in place.

Reply:
I will have the follow up PR for the command in kpt once this is merged so that I can point to kyaml master in kpt. We can release kpt after the command is merged.